tests/Type/Symfony/ExtensionTest.php

```diff
@@ -14,49 +14,49 @@ class ExtensionTest extends TypeInferenceTestCase
 	/** @return mixed[] */
 	public function dataFileAsserts(): iterable
 	{
-		yield from $this->gatherAssertTypes(__DIR__ . '/data/envelope_all.php');
-		yield from $this->gatherAssertTypes(__DIR__ . '/data/header_bag_get.php');
-		yield from $this->gatherAssertTypes(__DIR__ . '/data/response_header_bag_get_cookies.php');
+		yield from self::gatherAssertTypes(__DIR__ . '/data/envelope_all.php');
+		yield from self::gatherAssertTypes(__DIR__ . '/data/header_bag_get.php');
+		yield from self::gatherAssertTypes(__DIR__ . '/data/response_header_bag_get_cookies.php');
 
 		if (class_exists('Symfony\Component\HttpFoundation\InputBag')) {
-			yield from $this->gatherAssertTypes(__DIR__ . '/data/input_bag.php');
+			yield from self::gatherAssertTypes(__DIR__ . '/data/input_bag.php');
 		}
 
-		yield from $this->gatherAssertTypes(__DIR__ . '/data/tree_builder.php');
+		yield from self::gatherAssertTypes(__DIR__ . '/data/tree_builder.php');
 
-		yield from $this->gatherAssertTypes(__DIR__ . '/data/ExampleBaseCommand.php');
-		yield from $this->gatherAssertTypes(__DIR__ . '/data/ExampleOptionCommand.php');
-		yield from $this->gatherAssertTypes(__DIR__ . '/data/ExampleOptionLazyCommand.php');
-		yield from $this->gatherAssertTypes(__DIR__ . '/data/kernel_interface.php');
-		yield from $this->gatherAssertTypes(__DIR__ . '/data/request_get_content.php');
+		yield from self::gatherAssertTypes(__DIR__ . '/data/ExampleBaseCommand.php');
```

Comment:
These `$this->` to `self::`  are unrelated to the rest of the PR. Please remove them. If you want, you can submit them in a separate PR. Thanks.